resources/dicts/thoughts/alive/general.json

```
@@ -493,7 +501,13 @@
             "Is feeling proud of {PRONOUN/m_c/self}",
             "Is daydreaming about a Clan celebration in {PRONOUN/m_c/poss} honor",
             "Offers to help {PRONOUN/m_c/poss} Clanmates with their duties",
-            "Imagines how {PRONOUN/m_c/subject} would run the Clan better"
+            "Wonders how {PRONOUN/m_c/subject} would run the Clan better",
+            "Considers what {PRONOUN/m_c/poss} first actions as leader will be",
```

```suggestion
            "Considers what {PRONOUN/m_c/poss} first actions as leader would be",
```
maybe change this to 'would' so it doesn't sound like m_c is _definitely_ going to be leader?